.github/workflows/style-to-repo_trunk.yml

```diff
@@ -0,0 +1,15 @@
+name: Build and deploy style files
+on:
+  pull_request_target:
+    types:
+      - closed
```

Comment:
I think we should target every commit, since there might be changes in style code that are not processed via PRs.